Modules/_ctypes/_ctypes.c

```diff
@@ -3614,7 +3614,7 @@ _build_callargs(PyCFuncPtrObject *self, PyObject *argtypes,
         case (PARAMFLAG_FIN | PARAMFLAG_FOUT):
             *pinoutmask |= (1 << i); /* mark as inout arg */
             (*pnumretvals)++;
-            /* fall through to PARAMFLAG_FIN... */
+            /* fall through */
```

Comment:
gcc only recognizes "fall through" (and a couple of other versions), but not that comment with additional words.

I only changed comments that aren't recognized.